relay-server/src/actors/events.rs

```diff
@@ -1018,9 +1018,22 @@ impl EventProcessor {
                 // Minidump scrubbing can fail if the minidump cannot be parsed. In this case, we
                 // must be conservative and treat it as a plain attachment. Under extreme
                 // conditions, this could destroy stack memory.
-                if let Err(scrub_error) = processor.scrub_minidump(filename, &mut payload) {
-                    log::debug!("failed to scrub minidump: {}", LogError(&scrub_error));
-                    processor.scrub_attachment(filename, &mut payload);
+                let start = Instant::now();
+                match processor.scrub_minidump(filename, &mut payload) {
+                    Ok(modified) => {
+                        metric!(
+                            timer(RelayTimers::MinidumpScrubbing) = start.elapsed(),
+                            status = if modified { "ok" } else { "n/a" },
+                        );
+                    }
+                    Err(scrub_error) => {
+                        metric!(
+                            timer(RelayTimers::MinidumpScrubbing) = start.elapsed(),
+                            status = "error"
+                        );
+                        log::warn!("failed to scrub minidump: {}", LogError(&scrub_error));
+                        processor.scrub_attachment(filename, &mut payload);
```

Comment:
Can we also time this? Should be easier since this is infallible and the metric macro has block syntax

Reply:
I kind of wanted to time this inside `scrub_attachments` but that's in the general crate and doesn't have the metrics.  Now we'll have to remember to add this metric every time we call it :(